src/MSBuild/Resources/xlf/Strings.de.xlf

```diff
@@ -155,17 +155,17 @@ Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.
 
                      This flag is experimental and may not work as intended.
     </source>
-        <target state="new">  -graphBuild[:True|False]
-                     Causes MSBuild to construct and build a project graph.
+        <target state="translated">  -graphBuild[:True|False]
+                     Führt dazu, dass MSBuild ein Projektdiagramm konstruiert und erstellt.
 
-                     Constructing a graph involves identifying project
-                     references to form dependencies. Building that graph
-                     involves attempting to build project references prior
-                     to the projects that reference them, differing from
-                     traditional MSBuild scheduling.
-                     (Short form: -graph)
+                     Das Konstruieren eines Diagramms beinhaltet das Identifizieren von
+                     Projektverweisen auf Formularabhängigkeiten. Beim Erstellen des Diagramms
```

Comment:
@v-zbsail `Formularabhängigkeiten` is wrong here as a translation for `to form dependencies`, it's talking about _creating_ dependencies and not about a form in the "sheet of paper" sense. And same issue for Diagramm vs Graph that I mentioned in the other comment.

Better translation would be:
> Das Konstruieren eines Graphs beinhaltet das Identifizieren von Projektverweisen um Abhängigkeiten zu erstellen.

Reply:
@janbaltus perfect, thank you very much!